react-components/src/components/NodeCacheProvider/utils.ts

```diff
@@ -26,11 +27,20 @@ export function modelRevisionAssetIdsToKey(
   return `${modelRevisionId.modelId}/${modelRevisionId.revisionId}/${assetId}`;
 }
 
-export function getAssetIdOrExternalIdFromAnnotation(
+export function getAssetIdOrExternalIdFromPointCloudAnnotation(
   annotation: AnnotationModel
 ): string | number | undefined {
   return (
     (annotation.data as AnnotationsBoundingVolume).assetRef?.id ??
     (annotation.data as AnnotationsBoundingVolume).assetRef?.externalId
   );
 }
+
+export function getAssetIdOrExternalIdFromImage360Annotation(
+  annotation: AnnotationModel
```

Comment:
Just wondering - is there a chance we will be able to change the type of this field to `Image360AnnotationModel` and then update where it is used?

Reply:
I can look into it later 😄